pyfranca/ast.py

```diff
@@ -339,6 +345,15 @@ def __init__(self, name, key_type, value_type):
         self.value_type = value_type
 
 
+class Constant(ComplexType):
```

Comment:
simply exhange ComplexType with object result in a ValueError
"Unexpected namespace member type."

Currently I do  not know all the base classes you use very well. I check this.  If you already have found a solution, please tell me. I not sure the I understand what you mean with "a family of AST representations." Maybe you can get more into detail.